packages/image_picker/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerDelegate.java

```diff
@@ -368,6 +393,26 @@ private void handleChooseImageResult(int resultCode, Intent data) {
     finishWithSuccess(null);
   }
 
+  private void handleChooseMultiImageResult(int resultCode, Intent data) {
+    if (resultCode == Activity.RESULT_OK && data != null) {
+      ArrayList<String> paths;
```

Comment:
we can initialize the "paths" here because both if and else cause do it. 
So: `ArrayList<String> paths = new ArrayList<>();` and remove both init below.

Reply:
Done.